lib/task_3/task3.dart

```diff
@@ -0,0 +1,19 @@
+// Реализуйте метод, который принимает строку слов, разделённых пробелами.
+// Задача — найти в данной строке числа и вернуть коллекцию num этих чисел.
+
+List<int> numbersInString(String text) {
```

Comment:
```suggestion
List<num> numbersInString(String text) {
```

Числа могут быть и дробными. Думаю, не стоит ограничивать себя только целыми числами.